changelog/change_drop_ruby_24.md

```diff
@@ -1 +1 @@
-* [#9647](https://github.com/rubocop/rubocop/pull/9647): Drop support for Ruby 2.4. ([@koic][])
+* [#9648](https://github.com/rubocop/rubocop/pull/9648): (**Breaking**) Drop support for Ruby 2.4. ([@koic][])
```

Comment:
We agreed that we won't consider this a breaking change for the purposes of SemVer. Not sure if we should tag it as breaking here.

Reply:
Ah, exactly! I updated only to correct the PR number.